Gems/ROS2/Code/Include/ROS2/VehicleDynamics/VehicleInputControlBus.h

```diff
@@ -24,15 +24,27 @@ namespace ROS2::VehicleDynamics
 
         virtual ~VehicleInputControlRequests() = default;
 
+        //! Set target for the vehicle linear speed. It should be realized over time according to drive model.
+        //! @param speedMps is a linear speed in meters per second in evry direction.
+        virtual void SetTargetLinearSpeedV3(AZ::Vector3 speedMps) = 0;
+
         //! Set target for the vehicle linear speed. It should be realized over time according to drive model.
         //! @param speedMps is a linear speed in meters per second with the plus sign in the forward direction.
-        virtual void SetTargetLinearSpeed(float speedMps) = 0;
+        virtual void SetTargetLinearSpeed(float speedMpsX) = 0;
```

Comment:
is `X` referring to the `X` direction? Maybe update the param argument to clarify